src/h5web/explorer/Explorer.tsx

```diff
@@ -58,10 +88,11 @@ function Explorer(props: Props): JSX.Element {
 
       {tree.children && (
         <TreeView
+          level={0}
```

Comment:
Remnant from an initial attempt, which I kept because it now seems cleaner: the nesting level is now managed in `TreeView` instead of in the nodes (`TreeNode`).